plugins/github-actions/src/components/Cards/RecentWorkflowRunsCard.tsx

```diff
@@ -103,10 +109,7 @@ export const RecentWorkflowRunsCard = (props: {
               title: 'Commit Message',
               field: 'message',
               render: data => (
-                <Link
-                  component={RouterLink}
```

Comment:
odd that it had this component declaration in the first place; that's for sure redundant (the implementation will render itself as that if it's a local target automatically).